binding/web/README.md

```diff
@@ -151,18 +167,18 @@ or initialize using a base64 string:
 import rhinoContext from "${PATH_TO_BASE64_RHINO_CONTEXT}";
 import rhinoParams from "${PATH_TO_BASE64_RHINO_PARAMS}";
 
-const handle = await Rhino.fromBase64(
+const handle = await Rhino.create(
   ${ACCESS_KEY},
-  { label: "rhino_model", base64: rhinoContext },
+  { base64: rhinoContext },
   inferenceCallback,
-  rhinoParams,
+  { base64: rhinoParams },
   options // optional options
 )
 ```
```

Comment:
```
const handle = await Rhino.create(
  ${ACCESS_KEY},
  rhinoContext,
  inferenceCallback,
  rhinoModel,
  options // optional options
)
```
Suggestion: just use one init since we explained above?